{
  "comments": [
    {
      "key": {
        "uuid": "8b8fc161_c8c65886",
        "filename": "occ_status.cpp",
        "patchSetId": 6
      },
      "lineNbr": 297,
      "author": {
        "id": 1000008
      },
      "writtenOn": "2021-10-12T22:00:22Z",
      "side": 1,
      "message": "Oh, since you\u0027re not reading from the filesystem here, it won\u0027t return a system_error.  Instead you can catch sdbusplus::exception::exception, and then print e.what() in the trace.",
      "range": {
        "startLine": 297,
        "startChar": 17,
        "endLine": 297,
        "endChar": 38
      },
      "revId": "d80a03dcedaa23e930a279c6ddf775e783bfbe29",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4",
      "unresolved": true
    }
  ]
}